nanoFramework.Graphics.Core/System/Drawing/Point.cs

```diff
@@ -0,0 +1,183 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
```

Comment:
Not 100% if this is safe to use... our execution engine may not be able to honor this request...